### PR TITLE
WT-7425 Fix for the -C command line option

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

**Comment by etienneptl** [comment]
unit-test-bucket10 failed in the PR testing. It is being tracked by WT-7186

File: test/cppsuite/tests/run.cxx [hunk 1]
```diff
@@ -67,26 +67,20 @@ value_missing_error(const std::string &str)
 
 /*
  * Run a specific test.
- * config_name is the configuration name. The default configuration is used if it is left empty.
```
Comment (luke-pearson) [inline test/cppsuite/tests/run.cxx:70]: Comment formatting is weird here.

File: test/cppsuite/tests/run.cxx [hunk 2]
```diff
@@ -101,19 +95,21 @@ run_test(const std::string &test_name, const std::string &config_name = "")
...
-     * -C   : Configuration. Cannot be used with -f.
-     * -f   : Filename that contains the configuration. Cannot be used with -C.
+     * -C   : Configuration. Cannot be used with -f. If no specific test is specified to be run, the
+     * same coniguration will be used for all existing tests.
+     * -f   : Filename that contains the configuration. Cannot be used with -C. If no specific test
+     * is specified to be run, the same coniguration will be used for all existing tests.
      * -l   : Trace level.
      * -t   : Test to run. All tests are run if not specified.
      */
     for (int i = 1; (i < argc) && (error_code == 0); ++i) {
```
Comment (luke-pearson) [inline test/cppsuite/tests/run.cxx:110]: Can we print a usage string here if the user inputs incorrect options? I've wanted this myself when trying to figure out how to run our runner.

File: test/cppsuite/tests/run.cxx (hunk 1 above)
Comment (etienneptl) [reply test/cppsuite/tests/run.cxx:70]: I am describing the role of each arg. One arg per line.

File: test/cppsuite/tests/run.cxx [hunk 3]
```diff
@@ -65,28 +65,54 @@ value_missing_error(const std::string &str)
     test_harness::debug_print("Value missing for option " + str, DEBUG_ERROR);
 }
 
+void
+print_help()
```
Comment (luke-pearson) [inline test/cppsuite/tests/run.cxx:69]: Should we call print_help from `value_missing_errror`?

Comment (etienneptl) [reply test/cppsuite/tests/run.cxx:63]: Sure

File: test/cppsuite/tests/run.cxx [hunk 4]
```diff
@@ -97,6 +91,13 @@ print_help()
     std::cout << "\t-t Test name to be executed." << std::endl;
 }
 
+void
+value_missing_error(const std::string &str)
+{
+    test_harness::debug_print("Value missing for option " + str, DEBUG_ERROR);
+    std::cout << "Try './run -h' for more information." << std::endl;
```
Comment (luke-pearson) [inline test/cppsuite/tests/run.cxx:98]: I think these should both be in the same debug_print style format?

**Review by luke-pearson — APPROVED** [review]
LGTM.

**Pull request merged**